app/controllers/DetermineTaxIdController.scala

```diff
@@ -69,6 +69,7 @@ class DetermineTaxIdController @Inject() (
         case TaxRegime.Epaye => Redirect(routes.NotEnrolledController.notEnrolled)
         case TaxRegime.Vat   => Redirect(routes.NotEnrolledController.notVatRegistered)
         case TaxRegime.Sa    => Redirect(routes.NotEnrolledController.notSaEnrolled)
+        case TaxRegime.Sia   => Redirect(routes.NotEnrolledController.notSaEnrolled)
```

Comment:
this doesn't seem quite right - if the user didn't have a NINO then I'm assuming that there'd be some other new special kickout page we could point to here which we could just have as a dummy endpoint for now

Reply:
in the pattern match above here:
```scala
 case j: Journey.Stages.Started =>
        enrolmentService.determineTaxIdAndUpdateJourney(j, request.enrolments) -> j.taxRegime
``` 
I think you're gonna have to do something different if it's SIA since SIA doesn't have an enrolment. What you'd need to do is to make sure we're retrieving the NINO from auth and make it available in the `request` here and then use that as the tax id if it exists.